infra/image/run-tests.sh

```diff
@@ -131,16 +192,22 @@ done
 
 [ ${SPLITS} -eq 0 ] && [ ${#ENABLED_MODULES[@]} -eq 0 ] && [ ${#ENABLED_TESTS[@]} -eq 0 ] && die -u "No test defined."
 
-export STOP_CONTAINER FORCE_ENV STOP_VIRTUALENV ansible_interpreter
+export STOP_CONTAINER STOP_VIRTUALENV
 
 # Ensure $python is set
-[ -z "${python}" ] && python="python3"
+for py in "python3" "python"
```

Comment:
Should this also test platform and system python?

Reply:
fixed.